lib/manager.js

```diff
@@ -58,3 +58,20 @@ exports.AssetManager = class AssetManager {
 		return this.manifest.set(originalPath, actualPath, targetDir);
 	}
 };
+
+function generateFingerprint(filepath, data) {
+	let filename = path.basename(filepath);
+	let ext = filename.indexOf(".") === -1 ? "" : "." + filename.split(".").pop();
```

Comment:
Why are we not using `path.extname` here? (See potentially [related discussion](https://github.com/faucet-pipeline/faucet-pipeline-images/pull/77#discussion_r1997547920) for `faucet-static`/`static-images`.)

(Though we might reasonably opt not to risk changing the implementation here, due to backwards-compatibility concerns.)